Replace dynamic dependencies with fixed requirements

devdev-automation opened the pull request:
Description
---------------

Replace dynamic dependencies with values taken directly from requirements.txt

If this PR will fix an issue, please address it:
Fix #1498 1498

Requirements
---------------

- [ ] Add your name to `CONTRIBUTORS.md`
- [ ] If this is a new feature, then please add some additional information about it to `CHANGELOG.md`

devdev-automation committed:
Replace dynamic dependencies with fixed requirements

install_requires values taken directly from requirements.txt

shelld3v commented:
Hi @devdev-automation,

Thanks for you effort, but do you think it's better to just parse `requirements.txt` instead of re-typing from that file? So the next time we need to add a new package requirement we only need to make change to `requirements.txt`.

shelld3v committed:
Remove version requirements

devdev-automation commented:
As there appears to be an issue with the current CI checks (latest merges into main also fail) are you able to merge this so that pipx will function correctly @shelld3v ?

devdev-automation committed:
Refactor install_requires to read from requirements.txt

Updated the setup.py to dynamically parse requirements from requirements.txt and removed hardcoded install_requires.

devdev-automation committed:
Add source and license comments to setup.cfg

Added comments regarding the source and license information for the requirements file.
